bigtable-hbase-dataflow/src/test/java/com/google/cloud/bigtable/dataflow/CloudBigtableIOIntegrationTest.java

```diff
@@ -102,44 +116,78 @@ public void testEstimatedAndSplitForSmallTable() throws Exception {
   public void testEstimatedAndSplitForLargeTable() throws Exception {
```

Comment:
Can you clarify a bit?
The change in this PA is caching sampleRowKeys, one way to verify that the sampleRowKeys is loaded first and reused later is:

1. SampleRowKeys = GetSampleRowKeys();
2. getEstimatedSizeBytes()
3. splitIntoBundles()
4. assert same (SampleRowKeys, GetSampleRowKeys())